### PR TITLE
feat: transformer functions

### PR DESCRIPTION
**Pull request by sergelogvinov**
Add functions to template executer.

# Pull Request

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [ ] you linted your code (`make lint`)
- [ ] you linted your code (`make unit`)

> See `make help` for a description of the available targets.

### CODE REVIEW

File: pkg/talos/helper.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 
 func ipDescovery(nodeIPs []string, ifaces []network.AddressStatusSpec) (publicIPv4s, publicIPv6s []string) {
```
Comment (smira) [inline pkg/talos/helper.go:26]:
```suggestion
func ipDiscovery(nodeIPs []string, ifaces []network.AddressStatusSpec) (publicIPv4s, publicIPv6s []string) {
```

File: pkg/talos/helper.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ import (
...
 	for _, iface := range ifaces {
-		if iface.LinkName == "kubespan" || iface.LinkName == "lo" {
+		if iface.LinkName == "kubespan" || iface.LinkName == "lo" || strings.HasPrefix(iface.LinkName, "dummy") {
```
Comment (smira) [inline pkg/talos/helper.go:28]: probably also want to skip `siderolink` here?

**Review by smira — APPROVED** [review]

File: pkg/talos/helper.go (hunk 2 above)
Comment (sergelogvinov) [reply pkg/talos/helper.go]: Yep, thanks!

**Pull request merged**